gatsby/package.json

```diff
@@ -32,7 +32,7 @@
   },
   "dependencies": {
     "@nice-digital/browserslist-config": "^2.0.1",
-    "@nice-digital/design-system": "^1.4.0",
+    "@nice-digital/design-system": "^1.6.2-alpha.0",
```

Comment:
Let's release main NDS build and bump this before we merge

Reply:
Done